hw1/task04/task04.py

```diff
@@ -0,0 +1,28 @@
+"""
+Classic task, a kind of walnut for you.
+
+Given four lists A, B, C, D of integer values,
+compute how many tuples (i, j, k, l) there are such that
+A[i] + B[j] + C[k] + D[l] is zero.
+We guarantee, that all A, B, C, D have same length of N where 0 ≤ N ≤ 1000.
+"""
+
+from collections import defaultdict
+
+
+def check_sum_of_four(a: list, b: list, c: list, d: list) -> int:
+    """Count sum of four where sum = 0."""
+    s = defaultdict(int)
```

Comment:
Does name `s` represent the content of the variable?

Reply:
Agreed